packtools/sps/utils/xml_utils.py

```diff
@@ -350,13 +366,15 @@ def remove_subtags(node, tags_to_keep=None, tags_to_remove_with_content=None, ta
     all_tags_to_keep = _generate_tag_list(tags_to_keep, tags_to_convert_to_html)
 
     text = ' '.join(text.split())
-    if node.tag in all_tags_to_keep:
-        return f'<{node.tag}>{text}</{node.tag}>'
+    if tag in all_tags_to_keep:
+        if attribs:
+            return f'<{tag} {attribs}>{text}</{tag}>'
+        return f'<{tag}>{text}</{tag}>'
     return text
 
 
 def process_subtags(node, tags_to_keep=None, tags_to_remove_with_content=None, tags_to_convert_to_html=None):
-    std_to_keep = ['sup', 'sub', 'mml:math', 'math']
+    std_to_keep = ['sup', 'sub', 'mml:math', 'mml:mrow', 'mml:msqrt', 'mml:mn', 'math']
```

Comment:
@Rossi-Luciano Bastaria `mml:math` ou `math`, porque o que está dentro, preserva tudo. Se necessário adicione `tags_to_keep_with_content=None,`

Reply:
https://it.wikipedia.org/wiki/MathML

https://jats.nlm.nih.gov/publishing/tag-library/1.3/element/mml-math.html

O conjunto de mml:math é grande, não compensa adicionar como uma lista